pkg/util/shell/shell.go

```diff
@@ -90,7 +90,7 @@ func GetPrefixSuffixDelimiterForSet(userShell string, pathVar bool) (prefix, suf
 		prefix = "SET "
 		suffix = "\n"
 		if pathVar {
-			suffix = ";%PATH%" + suffix
+			suffix = ";%PATH%;" + suffix
```

Comment:
@gbraad , will test that.  But what if user try to use SET instructions himself instead of going through @FOR ... instructions ?